sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.controller.service.ts

```diff
@@ -338,4 +344,57 @@ export class EducationProgramOfferingControllerService {
       ),
     };
   }
+
+  /**
+   * Gets the offering simplified details, not including, for instance,
+   * validations, approvals and extensive data.
+   * Useful to have an overview of the offering details, for instance,
+   * when the user needs need to have quick access to data in order to
+   * support operations like confirmation of enrolment or scholastic
+   * standing requests or offering change request.
+   * @param offering to be transformed.
+   * @param options method options:
+   * - `locationId`: location for authorization.
+   * @returns education program offering.
+   */
+  async getOfferingById(
+    offeringId: number,
+    options?: {
```

Comment:
Can we please remove optional for options?. As the one reference that is using it use options.

Reply:
It is expected that the method will be consumed and also not provide the location id.
Please let me know if it would be considered a blocker.

Reply:
NO its not a blocker, I saw there was only one reference so suggested the change.